examples/pytorch/arma/README.md

```diff
@@ -0,0 +1,78 @@
+# DGL Implementation of ARMA
+
+This DGL example implements the GNN model proposed in the paper [Graph Neural Networks with convolutional ARMA filters](https://arxiv.org/abs/1901.01343). For the original implementation, see [here](https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#torch_geometric.nn.conv.ARMAConv).
```

Comment:
"For the original implementation, see [here](https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#torch_geometric.nn.conv.ARMAConv)."

Is this the implementation from the authors? If not, there's no need to mention it.

Reply:
They mentioned PyG public implementation before section 6.1, but this is not from the authors.

Reply:
I think you don't need to mention it as it's not from the authors.